app/controllers/api/v1/interactive_run_states_controller.rb

```diff
@@ -26,7 +27,11 @@ def update
         @interactive_run_state.learner_url = params['learner_url']
       end
       if params.has_key?('metadata')
-        @interactive_run_state.metadata = params['metadata']
+        # Note that metadata is always merged, not overwritten, as multiple apps and clients can set its own metadata.
```

Comment:
```suggestion
        # Note that metadata is always merged, not overwritten, as multiple apps and clients can set their own metadata.
```

Reply:
Thanks. I'm starting to think that you have some Gmail plugin that scans all your emails from Github and checks grammar :wink: